### DISCUSSION TITLE
[General] `eth_getFilterChanges` endpoint is completely broken in official RPC

### DISCUSSION THREAD

**Discussion by ouromoros** [zkSync-Community-Hub/zksync-developers#232] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

We've observed that even if we create a new filter id and then immediately call `get_filterChanges` in the same WebSocket connection. The RPC would respond with "Filter not found" error. As was mentioned in this Github issue (https://github.com/matter-labs/zksync-era/issues/87), the behavior was likely due to too many activities. Since the filter id expires in milli-seconds, it's reasonable to assume that some people are spamming the RPC with new filter calls, which results in the endpoint basically broken.

Is there any plan to fix it? This is a serious issue with the official RPC not functioning correctly.

### What is the specific issue or error you're encountering?

Left empty

### Can you share the error messages or logs you're receiving, if any?

Left empty

### Have you made any recent changes to the contract before encountering this issue?

Left empty

### Are there any external libraries or contracts that your contract interacts with?

Left empty

### Can you provide the relevant portions of your contract code where the issue is occurring?

Left empty

### Have you tried to isolate the problem, and if so, what were the results?

Left empty

### What steps have you already taken to try to resolve the issue?

Left empty

**Comment by MexicanAce** [comment]
Hi @ouromoros 👋 

Are you still seeing this issue? Over the last few weeks, the team has been working on improving the spam that's been overloading this endpoint.
